content/en/continuous_integration/intelligent_test_runner/javascript.md

```diff
@@ -27,32 +27,83 @@ Intelligent Test Runner is only supported in the following versions and testing
 * `cypress>=6.7.0`
   * From `dd-trace>=4.2.0`, `dd-trace>=3.23.0` or `dd-trace>=2.36.0`.
   * Instrument your web application with code coverage: see more in [Cypress setup](#cypress-setup)
+* If you're reporting data through the Datadog Agent, use v6.40+/v7.40+.
 
 ## Setup
 
+### Test Visibility Setup
+
 Prior to setting up Intelligent Test Runner, set up [Test Visibility for Javascript and Typescript][2]. If you are reporting data through the Agent, use v6.40+/v7.40+.
 
-To enable Intelligent Test Runner, set the following environment variables:
+### UI activation
+You or a user in your organization with "Intelligent Test Runner Activation" permissions must activate the Intelligent Test Runner on the [Test Service Settings][3] page.
+
+{{< img src="continuous_integration/itr_overview.png" alt="Intelligent test runner enabled in test service settings in the CI section of Datadog.">}}
+
+### Configuring the test runner environment
+
+{{< tabs >}}
+
+{{% tab "On-Premises CI Provider (Datadog Agent)" %}}
+
+#### Additional Datadog Agent configuration
+
+In addition to the [Datadog API Key][1], the [Datadog Application key][2] must be specified in the [Agent Configuration File][3]
+
+`app_key` (Required)
+: The [Datadog Application key][2] used to query the tests to be skipped.<br/>
+**Default**: `(empty)`
+
+`api_key` (Required)
+: The [Datadog API key][1] used to upload the test results.<br/>
+**Default**: `(empty)`
+
+After configuring the Datadog Agent, run your tests as you normally do:
+
+{{< code-block lang="shell" >}}
+NODE_OPTIONS="-r dd-trace/ci/init" DD_ENV=ci DD_SERVICE=my-javascript-app yarn test
+{{< /code-block >}}
+
+[1]: https://app.datadoghq.com/organization-settings/api-keys
+[2]: https://app.datadoghq.com/organization-settings/application-keys
```

Comment:
I'm starting to wonder if we should have this link to the [API and application keys docs](https://docs.datadoghq.com/account_management/api-app-keys/) instead of the app UI, but if we do link to the app, we should probably use the site, right, because if they're in US3, or EU, this link would be broken?

Reply:
this is handled automatically. See https://docs.datadoghq.com/continuous_integration/troubleshooting/#your-tests-are-instrumented-but-datadog-isnt-showing-any-data for example. If you switch your site and click on test runs (coming from https://github.com/DataDog/documentation/blob/master/content/en/continuous_integration/troubleshooting.md?plain=1#L212, which is hardcoded to datadoghq.com), it works.

Reply:
Didn't realize it was automatic. 😅

Reply:
I just learnt it today 😅